From c17307151d434981d566b3b60da298ab82e1cc07 Mon Sep 17 00:00:00 2001
From: Scott Ellis <scott@pansenti.com>
Date: Sat, 30 Nov 2013 10:55:27 -0500
Subject: [PATCH] Prep for debian build

---
 SyntroCam.pro | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/SyntroCam.pro b/SyntroCam.pro
index fc3cfbf..abf4b06 100644
--- a/SyntroCam.pro
+++ b/SyntroCam.pro
@@ -32,7 +32,7 @@ unix {
 		CONFIG += link_pkgconfig
     		LIBS += -lasound
                 PKGCONFIG += syntro
-		target.path = /usr/bin
+		target.path = $$PREFIX/bin
 		INSTALLS += target
         }
 }
-- 
1.8.1.2

