From 0d1fa475150ce9bc8b8c6ddf6603c332f3df12b3 Mon Sep 17 00:00:00 2001
From: Scott Ellis <scott@pansenti.com>
Date: Thu, 12 Dec 2013 08:38:46 -0500
Subject: [PATCH] Prep for debian package build

---
 SyntroLCam.pro | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/SyntroLCam.pro b/SyntroLCam.pro
index 3c05db8..41783d1 100644
--- a/SyntroLCam.pro
+++ b/SyntroLCam.pro
@@ -22,7 +22,7 @@ PKGCONFIG += syntro
 
 LIBS += -lasound
 
-target.path = /usr/bin
+target.path = $$PREFIX/bin
 
 INSTALLS += target
 
-- 
1.8.1.2

